### PR TITLE
Support --self-contained-interactive-authoring-enabled in integration-runtime self-hosted create

### PR DESCRIPTION
**Pull request by Jingshu918**
…-runtime self-hosted create

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/datafactory/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.11.0
++++++
+* Support --selfcontainediaenabled in integration-runtime self-hosted create
```
Comment (kairu-ms) [inline src/datafactory/HISTORY.rst:7]: Pleas split words in argument option name by '-'

Comment (Jingshu918) [reply src/datafactory/HISTORY.rst]: fixed

**Review by kairu-ms — APPROVED** [review]

**Review by kairu-ms — CHANGES_REQUESTED** [review]
The ‘enable’ should be the prefix of argument.

File: src/datafactory/azext_datafactory/generated/_params.py [hunk 2]
```diff
@@ -306,7 +306,13 @@ def load_arguments(self, _):
             help="The base definition of a linked integration "
             "runtime. Expected value: json-string/json-file/@json-file.",
             arg_group="Type Properties",
-        )
+        ),
+        c.argument(
+            "enableselfcontainedia",
+            options_list=['--enable-self-contained-interactive-authoring', '--scia-enabled'],
```
Comment (kairu-ms) [inline src/datafactory/azext_datafactory/generated/_params.py:312]: Same here for the short alias, please use enable as prefix

**Review by kairu-ms — APPROVED** [review]
LGTM

File: src/datafactory/azext_datafactory/manual/version.py [hunk 3]
```diff
@@ -8,4 +8,4 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.10.0"
+VERSION = "0.11.0"
```
Comment (evelyn-ys) [inline src/datafactory/azext_datafactory/manual/version.py:11]: We have new version guideline for cli extensions. Maybe you need to upgrade to `1.0.0` now. cc @jsntcy

Comment (Jingshu918) [reply src/datafactory/azext_datafactory/manual/version.py]: update

**Pull request merged**